Tensile/Tests/extended/global_split_u/hgemm_gsu_minkforgsu.yaml

```diff
@@ -0,0 +1,65 @@
+GlobalParameters:
+  PrintLevel: 1
+  ForceRedoBenchmarkProblems: True
+  ForceRedoLibraryLogic: True
+  ForceRedoLibraryClient: True
+  CMakeBuildType: Release
+  EnqueuesPerSync: 1
+  SyncsPerBenchmark: 1
+  LibraryPrintDebug: False
+  NumElementsToValidate: 128
+  ValidationMaxToPrint: 16
+  ValidationPrintValids: False
+  ShortNames: False
+  MergeFiles: True
+  PlatformIdx: 0
+  DeviceIdx: 0
+  #  DataInitTypeAB: 0
+  PrintSolutionRejectionReason: True
+
+BenchmarkProblems:
+  - # sgemm NN
+    - # ProblemType
+      OperationType: GEMM
+      DataType: H
+      DestDataType: H
+      ComputeDataType: S
+      HighPrecisionAccumulate: True
+      TransposeA: False
+      TransposeB: False
+      UseBeta: True
+      Batched: True
+
+    - # BenchmarkProblemSizeGroup
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - PackSummationDims: [0]
+        - NumLoadsCoalescedA: [1]
+        - NumLoadsCoalescedB: [1]
+        - WorkGroupMapping: [1]
+        - AssertFree1ElementMultiple : [4]
+        - AssertSizeLessThan: [{0: 5}]
+        - GlobalSplitUAlgorithm: ["MultipleBuffer"]
+        - GlobalSplitUWorkGroupMappingRoundRobin: [False]
+        - GlobalSplitUSummationAssignmentRoundRobin:  [True]
+        - EdgeType: ["ShiftPtr"]
+        - VectorWidth: [1]
+        - PrefetchGlobalRead: [false]
+        - StaggerU: [0]
+        - StaggerUStride: [0]
+        - StaggerUMapping: [0]
+        - MinKForGSU: [32,33,34,35]
```

Comment:
The tensile kernels with these values will be exactly same (except for label).
It is not effective to run the test with same kernels multiple times.
Instead, we should try different scenarios.
(1)enable larger GSU with this change (covered by this scenario)
(2) try smallest MinKForGSU (use GSU = K / MinKForGSU)
(3) reject case (K / GSU < MinKForGSU)
(4) reject case 2 with GSU=1 (using not default MinKForGSU + GSU=1)